kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/GeneralErrorCorrectionValidation.xml

```diff
@@ -1,25 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns:p="http://www.springframework.org/schema/p"
-       xmlns:aop="http://www.springframework.org/schema/aop"
-       xmlns:tx="http://www.springframework.org/schema/tx"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
-                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
-                           http://www.springframework.org/schema/tx
-                           http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
-                           http://www.springframework.org/schema/aop
-                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
 
-    <bean id="GeneralErrorCorrection-routeDocumentValidation" parent="GeneralErrorCorrection-routeDocumentValidation-parentBean" scope="prototype">
+	<bean id="GeneralErrorCorrection-routeDocumentValidation" parent="GeneralErrorCorrection-routeDocumentValidation-parentBean" scope="prototype">
 		<property name="validations">
 			<list merge="true">
 				<bean parent="GeneralErrorCorrection-errorCertificationValidation" scope="prototype"/>
 				<bean parent="GeneralErrorCorrection-errorCertificationTabValidation" scope="prototype"/>
 			</list>
 		</property>
 	</bean>
-    
+
     <bean id="GeneralErrorCorrection-addAccountingLineValidation" parent="GeneralErrorCorrection-addAccountingLineValidation-parentBean" scope="prototype">
         <property name="validations">
             <list>
```

Comment:
Both have been corrected (again, missed due to being new in v6). Also, I am saving up all of these changes to push under one commit, so as to keep the git log a bit cleaner.